_includes/markdown/CSS.md

```diff
@@ -4,377 +4,578 @@ At 10up, we value content and the experience users will have reading it. We writ
 
 ```css
-/* Select list item 4 to 8, included */
-li:nth-child(n+4):nth-child(-n+8) {
-	color: red;
+:where(h1, h2, h3, h4, h5, h6) {
+  margin-block: 0;
 }
 ```
 
-### Network Requests
+### Avoid Excessive Nesting
 
-* Limit the number of requests by concatenating CSS files and encoding sprites and font files to the CSS file.
-* Minify stylesheets
-* Use GZIP compression when possible
-Automate these tasks with a PHP or/and JavaScript build process.
+Limit the depth of nesting in your CSS selectors. Two levels of nesting are acceptable, and three should only be used when absolutely necessary. Excessive nesting can make CSS harder to read and maintain. It also increases specificity unnecessarily, which complicates overriding styles and debugging. Over-nesting complicates maintenance and increases specificity unnecessarily.
 
-### CSS Specificity
+It’s okay to nest pseudo classes that represent state, for example:
 
-Stylesheets should go from less specific rules to highly specific rules. We want our selectors specific enough so that we don't rely on code order, but not too specific so that they can be easily overridden.
+```css
+/* block */
+.button {
 
-For that purpose, **classes** are our preferred selectors: pretty low specificity but high reusability.
+  /* ✅ state inside the block is fine */
+  &:hover {
+    /* styles here */
+  }
+}
 
-Avoid using `!important` whenever you can.
+/* ✅ element and modifier stay flat */
+.button__icon {
+  /* styles here */
+}
 
-Use [efficient selectors](https://csswizardry.com/2011/09/writing-efficient-css-selectors/).
+.button--primary {
+  /* styles here */
+}
+```
 
-Avoid:
+Not necessary to nest elements or modifiers:
 
 ```css
- div div header#header div ul.nav-menu li a.black-background {
-  background: radial-gradient(ellipse at center,  #a90329 0%,#8f0222 44%,#6d0019 100%);
+/* block */
+.button {
+
+  /* state inside the block is fine */
+  &:hover {
+    /* styles here */
+  }
+
+  /* ❌ element and modifier must not be nested inside the block */
+  & .button__icon {
+    /* styles here */
+  }
+
+  & .button--primary {
+    /* styles here */
+  }
 }
 ```
 
-### Inheritance
+### Avoid IDs
+
+Refrain from using IDs for styling purposes. IDs have high specificity, which makes them difficult to override and maintain. Instead, use classes for consistent and reusable styling.
+
+### Avoid \!important
+
+Reserve \!important for truly exceptional cases. Overusing \!important disrupts the cascade and makes debugging more difficult. Focus on writing clean, organized selectors instead of relying on this property to enforce styles.
```

Comment:
```suggestion
Reserve `\!important` for truly exceptional cases. Overusing `!important` disrupts the cascade and makes debugging more difficult. Focus on writing clean, organized selectors instead of relying on this property to enforce styles.
```